c2rust-transpile/src/translator/structs.rs

```diff
@@ -296,7 +343,7 @@ impl<'a> Translation<'a> {
         struct_id: CRecordId,
         field_ids: &[CDeclId],
         platform_byte_size: u64,
-    ) -> TranslationResult<(Vec<Field>, bool)> {
+    ) -> TranslationResult<(Vec<Field>, bool, bool)> {
```

Comment:
We probably want this `(Vec<Field>, bool, bool)` to be a `struct` with named fields at this point.